plugins/CoreHome/javascripts/popover.js

```diff
@@ -142,7 +142,17 @@ var Piwik_Popover = (function () {
 
         /** Set the title of the popover */
         setTitle: function (titleHtml) {
+            var titleText = $('<div>' + titleHtml + '</div>').text();
+            if (titleText.length > 60) {
+                titleHtml = $('<span>').attr('class', 'tooltip').attr('title', titleText).html(titleHtml);
```

Comment:
I don't tink so. Setting the title using `attr` should escape it. Tried it with a title including `"`